### ISSUE TITLE
chat answer CSS is not the same as in the playground/chatgpt itself

### ISSUE THREAD

**Issue by motibenm** [microsoft/sample-app-aoai-chatGPT#117]
Hey,
We noticed that the Azure OpenAI deployment answer is not coloring code like the normal chatgpt/the playground version in azure ai.
Is there an option to fix this somehow?
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/140501804/3dd926e1-d481-4ddf-9c77-4855cfb87359)
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/140501804/dea6fa6d-e21a-4dbd-b98e-95cfc3d519d4)

**Comment by marwic-norlys** [comment]
Agree. Its a pitty this is not the same - as the one in playground looks way better + has more features (as markdown).

**Comment by motibenm** [comment]
I did manage to do make some sort of coloring.
You need to install highlight js in your project, and then in the Chat.tsx, enter the following 2 lines at the top -
import hljs from "highlight.js";
import "highlight.js/styles/a11y-dark.css";

and this with the other useEffects -
    useEffect(() => {
        hljs.highlightAll();
    },[isLoading]);

**Comment by marwic-norlys** [comment]
Thanks. Still hoping MS will do this for us soon 👍
